src/pages/account/mijn-historiek/index.tsx

```diff
@@ -27,12 +96,54 @@ const AccountMyHistory: NextPage = () => {
 				className="p-account-my-history"
 				contentTitle={t('pages/account/mijn-historiek/index___mijn-historiek')}
 			>
-				<div className="l-container">History table</div>
+				{(visits.data?.items?.length || 0) > 0 ? (
+					<div className="l-container l-container--edgeless-to-lg">
+						<Table
+							className="u-mt-24"
+							options={
+								// TODO: fix type hinting
+								/* eslint-disable @typescript-eslint/ban-types */
+								{
+									columns: HistoryTableColumns({ t }) as Column<object>[],
+									data: visits.data?.items || [],
+									initialState: {
+										pageSize: HistoryItemListSize,
+										sortBy: sortFilters,
+									},
+								} as TableOptions<object>
+								/* eslint-enable @typescript-eslint/ban-types */
+							}
+							onSortChange={onSortChange}
+							sortingIcons={sortingIcons}
+							pagination={({ gotoPage }) => {
+								return (
+									<PaginationBar
+										className="u-mt-16 u-mb-16"
+										count={HistoryItemListSize}
+										start={Math.max(0, filters.page - 1) * HistoryItemListSize}
```

Comment:
Would this be more logical for the user to show:
1 - 2 of the 2
instead of
0 - 2 of the 2

![image](https://user-images.githubusercontent.com/1710840/158380897-f414e0e1-1549-4351-a272-55b7a6da02c6.png)